app/code/Magento/InventoryInStorePickupFrontend/Block/Checkout/LayoutProcessor.php

```diff
@@ -97,6 +90,34 @@ private function removeStorePickup(array $jsLayout): array
         return $jsLayout;
     }
 
+    /**
+     * Add ui store pickup components to layout.
+     *
+     * @param array $jsLayout
+     * @return array
+     */
+    private function addStorePickupComponents(array $jsLayout): array
+    {
+        $jsLayout = $this->arrayManager->merge(
+            $this->arrayManager->findPath('store-pickup', $jsLayout),
+            $jsLayout,
+            [
+                'config' => [
+                    'nearbySearchRadius' => $this->getSearchRadius(),
+                ],
+            ]
+        );
+        $jsLayout = $this->arrayManager->merge(
```

Comment:
Please, define this config in layout.xml file

Reply:
@sidolov, done